get-involved.md

```diff
@@ -4,46 +4,47 @@ title: Get Involved
+    - [Publicly-viewable Kanban board](https://wekan.tomesh.net/b/LWS8X7sGFXqDgZ7ag/tomesh-net) keeps track of our many projects
+    - [GitHub](https://github.com/tomeshnet) hosts our open-source software and all important documents
 
 ## How do I work with you?
 {:#work-with-us}
 
-### Collaborate on our Kanban Board at [wekan.tomesh.net](https://wekan.tomesh.net)
-Toronto Mesh uses a [Kanban (看板)](https://en.wikipedia.org/wiki/Kanban) to keep track of ideas and progress. The software we use is the self-hosted open-source [Wekan](https://github.com/wekan/wekan). Anyone is free to create an account and participate.
-
-You can contribute to the project in many ways. Start by [creating an account](https://wekan.tomesh.net) for our Kanban, and then ask on {% include icon-matrix.html %} or email {% if site.email %}{% include icon-email.html %}{% endif %} for an invite to our main board.
+Toronto Mesh uses a [Kanban (看板)](https://en.wikipedia.org/wiki/Kanban) to keep track of ideas and progress. The software we use is the self-hosted open-source [Wekan](https://github.com/wekan/wekan). Create an account on [wekan.tomesh.net](https://wekan.tomesh.net), and then ask on {% include icon-matrix.html %} or email {% if site.email %}{% include icon-email.html %}{% endif %} for an invite to our board.
 
 #### Propose an Idea
 
 - Create a new card in the **Ideas** column by adding a short title
-- Include details in the description
-- Add appropriate labels for easy filtering
+- Include details in the description and add appropriate labels for easy filtering
 
 #### Participate in the Discussion
 
-- You can add a comment to an existing card, regardless of what column the card is in
-- Visit [tomesh.net/events]({{ site.baseurl }}/events) and our {% include icon-matrix.html %} chat channel to talk with us
+- You can add a comment to any card
+- Discuss a card on {% include icon-matrix.html %}
 
 #### Work on a Card
 
 - Find a card you would like to take ownership and add yourself as a member to that card
-- Move the card to the **In Progress** column, or **Discuss** column if that needs to happen first for the particular card
-- When completed, move the card to the **Done** column (typically for a task that leads to a Pull Request in one of our [GitHub repositories](https://github.com/tomeshnet/) is usually considered done when reviewed and merged)
+- Move the card to the **Discuss** or **In Progress** column, then **Done** when completed
+
+### Contribute to our Git Repositories
+
+We use git, a version control tool, and GitHub to work on this [website](https://github.com/tomeshnet/tomesh.net), our [Raspberry Pi prototype](https://github.com/tomeshnet/prototype-cjdns-pi2), and keep record of [important documents](https://github.com/tomeshnet/documents) like our meeting notes. Please see our [git workflow](https://github.com/tomeshnet/documents/blob/master/CONTRIBUTING.md) and contribute on [GitHub](https://github.com/tomeshnet/).
+
```

Comment:
Maybe we can link to this repo too?  It has a lot of really great slide shows - just thinking about our convo in Dec about moving this info up in visibility:  https://github.com/tomeshnet/tomesh.101